src/api/java/com/minecolonies/api/blocks/AbstractBlockHut.java

```diff
@@ -6,6 +6,7 @@
 import com.ldtteam.structurize.util.PlacementSettings;
 import com.minecolonies.api.IMinecoloniesAPI;
 import com.minecolonies.api.MinecoloniesAPIProxy;
+import com.minecolonies.api.blocks.interfaces.IBuildingBrowsableBlock;
```

Comment:
imports in this file?

Reply:
That's the hotfix for the 1.19 merge that you broke 😛  Feel free to apply that commit separately to get that working faster if you want to delay merging this PR.